_docs/reference/config/mixer/attribute-vocabulary.md

```diff
@@ -63,4 +63,8 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | connection.sent.bytes_total | int64 | Total number of bytes sent by a destination service during the lifetime of a connection. | |
 | connection.duration | duration | The total amount of time a connection has been open. | |
 | context.protocol | string | Protocol of the request or connection being proxied. | tcp |
-| context.time | timestamp | The timestamp of the Report() call. | |
\ No newline at end of file
+| context.time | timestamp | The timestamp of the Report() call. | |
```

Comment:
@douglas-reid Can we find a better description for context.time? In particular, I'd like to not mention the Report() call here. This is an attribute, it can be used in cases other than Report. For example, it could be used in Check and could be used somewhat by a service calling Mixer.

Reply:
sure :). it is meant to be distinct from both `request.time` and `response.time`. It covers TCP long-lived connection scenarios (as well as batching scenarios) to allow tracking of when the client called Mixer. I'm sure it can be used in other scenarios too.

Would "timestamp of Mixer operation" be better?

Reply:
Yeah, I think that would be an improvement. Chengliang, please update as part of this PR.

Reply:
Done.